src/engraving/dom/select.cpp

```diff
@@ -620,6 +620,10 @@ void Selection::updateSelectedElements()
                         appendFiltered(el);
                     }
                 }
+                Tuplet* tuplet = cr->tuplet();
```

Comment:
Does this also work for nested tuplets?

Reply:
Alas, no

Reply:
I _think_, this should be easy enough to accomodate (famous last words). Looking into it.

Reply:
@cbjeukendrup or @RomanPudashkin - latest version should accomodate nested tuplets